Update how-to-change-password.md (#2230)
CSCfi/csc-user-guide: docs/accounts/how-to-change-password.md

# How to change password

> You're required to change your CSC user account password annually. You will receive notifications from CSC when your password is about to expire. If you fail to change the password in time your CSC user account will be temporarily disabled you must contact CSC Servicedesk to have it reopened. 

Reviewer: ```suggestion
You are required to change your CSC user account password annually. You will receive notifications from CSC when your password is about to expire. If you fail to change the password in time, your CSC user account will be temporarily disabled and you must contact CSC Service Desk to have it reopened. 
```
